functional-tests/bdd-tests/TestBaseLicenceRenewal.cs

```diff
@@ -448,23 +448,16 @@ public void RenewLicence(string responses)
             ClickLicencesTab();
 
             // reload Licences page as needed
+            
             for (int i = 0; i < 5; i++)
             {
-
-                try
-                {
-                    if (ngDriver.FindElement(By.XPath("//body[contains(.,'Active')]")).Displayed)
-                    {
-                        break;
-                    }
-                }
-                catch (Exception) 
+                if ((ngDriver.FindElement(By.XPath("//body[contains(.,'Active')]")).Displayed) == false)
                 {
                     ngDriver.Navigate().Refresh();
                     System.Threading.Thread.Sleep(2000);
-                } 
+                }
```

Comment:
Can you add an else { break; } to this - as otherwise we will always do this 5 times.

Also, it would be a good idea to surround this with a try catch just in case the FindElement fails.